README.md

```diff
@@ -8,62 +8,21 @@ This is the frontend application for [Insights Remediations](https://github.com/
 
 ### Running the application
 
-#### CI stable env
+1. install and start the dev server in an environment of your choice:
 
-1. install and start the dev server
 ```sh
 npm i
-npm run start:proxy
+npm run start
 ```
-2. Open browser at [https://ci.foo.redhat.com:1337/insights/remediations](https://ci.foo.redhat.com:1337/insights/remediations)
+While you run the command above, you will be prompted to choose an enviroment.
 
-#### CI beta env
-
-1. install and start the dev server
-```sh
-npm i
-npm run start:proxy:beta
-```
-2. Open browser at [https://ci.foo.redhat.com:1337/preview/insights/remediations](https://ci.foo.redhat.com:1337/preview/insights/remediations)
-
-#### Using other environments
-
-1. open `dev.webpack.config.js`
-2. add `env` attribute with required value to the `webpackProxy` object.
-
-For example, for prod-beta add following:
-```js
-{
-  env: 'prod-beta'
-}
-```
- and run `npm run start:proxy:beta` (the `:beta` suffix configures webpack to serve assets at /preview location.)
+2. Open browser at [https://stage.foo.redhat.com:1337/insights/remediations]( https://stage.foo.redhat.com:1337/insights/remediations)
```

Comment:
Or possibly at URL with `/preview`

Reply:
Makes sense. As the `fec dev` generates a dynamic URL, I have made the wording more abstract, but useful